build.gradle

```diff
@@ -40,10 +40,6 @@ allprojects {
         google()
         mavenCentral()
         jcenter()
-        maven {
-            url 'https://maven.google.com/'
-            name 'Google'
-        }
```

Comment:
The reason this was removed is that it's duplicated with the one in line 40

Reply:
Thanks for pointing this out. We will include in the update PR.